### ISSUE TITLE
Domain specific features (optics specifically and everything else)

### ISSUE THREAD

**Issue by Krastanov** [sympy/sympy-paper#68]
I see that there is already an incomplete section dedicated to Optics. Is this staying in? I can work on description for some of the smaller submodules like that (those that I have worked on, but I will need to check how outdated my knowledge is).

**Comment by certik** [comment]
Yes, that would be great. We can discuss specifics on a case by case basis.

**Comment by moorepants** [comment]
Yes, please add something there. But keep in mind that the physics section may be one of the first things to be cut. I'm not sure how things are going to play out wrt to the length of the paper and what we want to focus on.
